<ui version="4.0" >
 <author>Frédéric Lambert &lt;konkistadorr@gmail.com></author>
 <class>KPlato::ChartPanelBase</class>
 <widget class="QWidget" name="KPlato::ChartPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>411</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="chartFrame" >
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="leftMargin" >
         <number>2</number>
        </property>
        <property name="topMargin" >
         <number>2</number>
        </property>
        <property name="rightMargin" >
         <number>2</number>
        </property>
        <property name="bottomMargin" >
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="curve2" >
          <property name="whatsThis" >
           <string>Budgeted Cost of Work Performed</string>
          </property>
          <property name="text" >
           <string>Budgeted Cost of Work Performed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="curve1" >
          <property name="whatsThis" >
           <string>Budgeted Cost of Work Scheduled</string>
          </property>
          <property name="text" >
           <string>Budgeted Cost of Work Scheduled</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="curve3" >
          <property name="whatsThis" >
           <string>Actual Cost of Work Performed</string>
          </property>
          <property name="text" >
           <string>Actual Cost of Work Performed</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources/>
 <connections/>
</ui>
